dashboard/app/helpers/users_helper.rb

```diff
@@ -82,6 +82,13 @@ def summarize_trophies(script, user = current_user)
       user_data[:trophies] = summarize_trophies(script, user)
     end
 
+    if script.professional_learning_course
+      unit_assignment = Plc::EnrollmentUnitAssignment.find_by(user: user, plc_course_unit: script.plc_course_unit)
+      if unit_assignment
+        user_data[:focusAreas] = unit_assignment.plc_module_assignments.map{ |a| a.plc_learning_module.stage.position }
```

Comment:
Ah good call, that makes sense.  Renamed `focusAreas` -> `focusAreaPositions` in 1ad78c8d1599de2004d6993570fbaae664a17fa7.